src/components/custom/default/ListHit.tsx

```diff
@@ -0,0 +1,109 @@
+import PropTypes from "prop-types";
+import {useMemo} from "react";
+import {Pill} from "@performant-software/core-data/ssr";
+
+interface Props {
```

Comment:
Should we factor this out somewhere common if it's used for multiple components?